Kitodo/src/main/java/de/sub/goobi/forms/BasisForm.java

```diff
@@ -33,6 +33,7 @@
     protected String filter = "";
     protected User user;
     protected String sortierung = "prozessAsc";
+    static final String redirectParameter = "faces-redirect=true";
```

Comment:
This is similar to a constant value. Why you are not writing this according to the coding and check style guidelines?